.github/workflows/reusable-build.yml

```diff
@@ -192,6 +192,7 @@ jobs:
           REACT_APP_GP2_AUTH0_DOMAIN: ${{ steps.setup.outputs.gp2-auth0-domain }}
           REACT_APP_GP2_SENTRY_DSN: ${{ steps.setup.outputs.react-app-gp2-sentry-dsn }}
           REACT_APP_GTM_CONTAINER_ID: ${{ steps.setup.outputs.react-app-gtm-container-id }}
```

Comment:
I think this should be renamed to REACT_APP_CRN_GTM_CONTAINER_ID to reflect that it is CRN only.

Reply:
updated